supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -209,7 +209,7 @@ You can use a TGT to request further access, and provide the user with a Single
 [discrete]
 [[tls]]
 ==== image:images/yes.png[yes] TLS (noun)
-*Description*: TLS is an initialism for Transport Layer Security (TLS), and it is the successor to the Secure Sockets Layer (SSL) protocol. Do not expand the abbreviation on the first usage.
+*Description*: _TLS_ is an initialism for "Transport Layer Security (TLS)", and it is the successor to the Secure Sockets Layer (SSL) protocol. Do not expand the abbreviation on the first usage.
```

Comment:
Would a statement referring to use as an initialism or an abbreviation refer to the "use" of a term?

Reply:
That's a good question. I'm not really sure, but I think it could make sense in that way. I'll update throughout.

Reply:
@dfitzmau I updated so that anywhere I say that something is an acronym/initialism/abbreviation, it uses quotes. Can you check your open PRs to make sure you're following the same too?